test/libsolidity/semanticTests/viaYul/cleanup/comparison.sol

```diff
@@ -0,0 +1,41 @@
+contract C {
+	function eq() public pure returns (bool) {
+		uint8 x = 1; uint8 y;
+		assembly { y := 0x0101 }
+		return (x == y);
+	}
+	function neq() public pure returns (bool) {
+		uint8 x = 1; uint8 y;
+		assembly { y := 0x0101 }
+		return (x != y);
+	}
+	function geq() public pure returns (bool) {
+		uint8 x = 1; uint8 y;
+		assembly { y := 0x0101 }
+		return (x >= y);
+	}
+	function leq() public pure returns (bool) {
+		uint8 x = 2; uint8 y;
+		assembly { y := 0x0101 }
+		return (x <= y);
+	}
+	function gt() public pure returns (bool) {
+		uint8 x = 2; uint8 y;
+		assembly { y := 0x0101 }
+		return (x > y);
+	}
+	function lt() public pure returns (bool) {
+		uint8 x = 1; uint8 y;
+		assembly { y := 0x0101 }
+		return (x < y);
+	}
+}
+// ====
+// compileViaYul: also
+// ----
+// eq() -> true
+// neq() -> false
+// leq() -> false
```

Comment:
The order of calls being different from the order of the function definitions made it confusing when I was looking through :p

Reply:
Fixed :-). I just checked that an version of ``isoltest`` before this PR has all of them fail and the new one has them pass :-).